### ISSUE TITLE
Update templates to use Fable 3.7.22

### ISSUE THREAD

**Issue by MangelMaxime** [SAFE-Stack/SAFE-template#561]
As [announce yesterday](https://twitter.com/FableCompiler/status/1656699611371786240), Fable 3.7.22 has been released.

It focus on improving the transition period due to Fable 4 and .NET 7.

Even if your template already use a `global.json` I think it is a nice idea to update the template to Fable 3.7.22 to benefit from the error report when failing to load a plugin. 

Indeed, before this release if you updated Feliz to v2 and compile your application with Fable 3 you would not get any error but the compiled code would not do what you want. Plugin is not applied and you are not notified of it, meaning you will get React runtime errors about hooks being only usable inside of a React component / function component.

**Comment by isaacabraham** [comment]
Yes, I support upgrading to this - it's a low risk change to the template
